frontend/templates/product/sections/ProductOverviewSection/ProductDescription.tsx

```diff
@@ -95,27 +96,12 @@ type VariantDescriptionProps = {
 
 function VariantDescription({ children }: VariantDescriptionProps) {
    return (
-      <Box
-         dangerouslySetInnerHTML={{
-            __html: children,
-         }}
+      <PrerenderedHTML
+         html={children}
          fontSize="sm"
          sx={{
-            ul: {
-               my: 3,
-               pl: 5,
-            },
```

Comment:
I think the styles here are different from the `PrerenderedHTML` styles. Does the rendered text look decent to you?

Reply:
reverted differing styles in [`6eda824`](https://github.com/iFixit/react-commerce/pull/1946/commits/6eda824d4e1efde6fd5d96129dc768329aa242fd)